wiki/en/Running-a-Server.md

```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
+
+For turning recording on or off (depending on the current state):
+
+~~~
+ [Unit]
+ Description=Toggle recording state of Jamulus Server
+ Requisite=Jamulus-Server
+
+ [Service]
+ Type=oneshot
+ ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
+~~~
+
+For starting a new recording:
+
+~~~
+ [Unit]
+ Description=Start a new recording on Jamulus Server
+ Requisite=Jamulus-Server
+
+ [Service]
+ Type=oneshot
+ ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
+~~~
+
+_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
+
+Run `sudo systemctl daemon-reload` to register them for first use.
+
+Now you can run these with the `systemctl` command, for example:
+
+`sudo systemctl start jamulusTogglerec` (assuming you named your unit file `jamulusTogglerec.service`)
+
+You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
+
+### Making a Server status page
+
+With the `-m` command line argument, Server statistics can be generated to be put on a web page.
+
+Here is an example php script using the Server status file to display the current Server status on a html page (assuming the following command line argument to be used: `-m /var/www/stat1.dat`):
+
+~~~
+<?php
+function loadserverstat ( $statfilename )
+{
+   $datei = fopen ( $statfilename, "r" );
+   while ( !feof ( $datei ) )
+   {
+          $buffer = fgets ( $datei, 4096 );
+          echo $buffer;
+   }
+   fclose($datei);
+}
+?>
+<?php loadserverstat ( "stat1.dat" ); ?>
+~~~
+
+
+# Servers on the desktop
+
+Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
 
 * **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
 
 * **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
 
-* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Ubuntu and related distros) and type `jamulus -s`. Hit return and you should see the Server control window. You can stop the Server by closing the Server window, or by using `CTRL+C` in the terminal. _(To run a headless Server [read this guide](Server-Linux))_
+* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Debian and related distros) and type `jamulus -s`. Hit return and you should see the Server control window. You can stop the Server by closing the Server window, or by using `CTRL+C` in the terminal. _(To run a headless Server [read this guide](#installing-and-running-a-server))_
```

Comment:
... or run the newly created desktop shortcut. Basically it's similar to Windows.